source/includes/steps-create-agg-pipeline.yaml

```diff
@@ -64,5 +64,17 @@ content: |
 
          .. figure:: /images/compass/agg-builder-exclude-stage-example.png
             :alt: Aggregation Builder exclude stage example
+---
+title: Optional. Use custom collation.
+level: 4
+ref: use-custom-collation
+content: |
+   a. Click the :guilabel:`Expand` icon to the left of the
+      :guilabel:`Folder` icon.
+
+      .. figure:: /images/compass/agg-builder-collation-stage-example.png
+            :alt: Aggregation Builder collation stage example
+
+   b. Enter the desired :manual:`collation </reference/collation/>`.
```

Comment:
Even though we link here, it might be worth providing a one-sentence brief on what collation is. Do you think that would be helpful enough to warrant doing?

Reply:
Sure.  Added the first sentence for the link target. Fixed.